vars/workers.groovy

```diff
@@ -57,6 +57,19 @@ def base(Map params, Closure closure) {
       // Try to clone from Github up to 8 times, waiting 15 secs between attempts
       retryWithDelay(8, 15) {
         scmVars = checkout scm
+
+        def mergeBase
+        if (env.ghprbTargetBranch) {
+          sh(script: "cd kibana && git fetch origin ${env.ghprbTargetBranch}")
```

Comment:
Can you add `label`s to these `sh` calls? I've been trying to be good about adding labels everywhere so that steps don't show up as just "Shell Script"